org/org.yaml

```diff
@@ -65,20 +66,23 @@ orgs:
     - popcor255
     - pradeepitm12
     - praveen4g0
+    - pratap0007
     - pritidesai
+    - PuneetPunamiya
     - raffamendes
+    - savitaashture
     - sbwsg
     - sclevine
+    - sergetron
     - shashwathi
     - simonjjones
     - skaegi
     - skim1420
-    - sergetron
+    - sm43
```

Comment:
@vdemeester you mentioned adding chetan-rns, pratap007 and PuneetPunamiy, did you mean to add sm43 as well?

Reply:
ah right, I need to update my commit :sweat_smile: